tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -54,7 +48,22 @@ class CommandLineDrawingTestConfigImpl : public DrawingTestConfigImpl {
 
 public:
     CommandLineDrawingTestConfigImpl(int argc, char** argv)
```

Comment:
the TestConfig base class is pure virtual so that we don't need to clowncar all of the TAEF behaviours into the "command line" one ;P